src/bitcoin/P2tr.mo

```diff
@@ -0,0 +1,33 @@
+import Types "./Types";
+import Ecdsa "../ecdsa/Ecdsa";
+import Script "./Script";
+import Segwit "../Segwit";
+import Result "mo:base/Result";
+
+module {
+    type PublicKey = Ecdsa.PublicKey;
+    type Script = Script.Script;
+
+    public type P2trKeyAddress = Types.P2trKeyAddress;
+    public type DecodedAddress = {
+        network : Types.Network;
+        publicKeyHash : [Nat8];
+    };
+
+    // Create script for the given P2TR key spend address.
```

Comment:
I'd find it helpful if the docu pointed to the respective spec/BIP (including the relevant section).

Reply:
Not sure if there's a specific section about this. Added a reference to BIP341.